inst/@sym/sym.m

```diff
@@ -693,3 +699,9 @@
 %! else
 %!   delete([myfile '.mat'])
 %! end
+
+%!test
+%! a = sym(e);
```

Comment:
This `sym(e)` test will fail on Matlab: needs wrapped in the `if exist OCTAVE_VER` construct.